### PR TITLE
Filters mobile

### PR DESCRIPTION
**Pull request by austensen**
This PR adds support for mobile to the filter bar. Makes the basic changes to the styles of filter components and the bar, plus adds the fullscreen wrapper of all filters - and the scroll snapping and shadow effects. Plus a small fix to the handling of min-max errors, and its default values I caught while in here.

Also included is an update to the filter input styling on desktop and mobile so they read more clearly as inputs (not full width, white background)

Not included is the mobile-specific behavior of applying in-progress filters when the user clicks outside of that filter's dropdown. It seems like this will require some more substantial reorganization of how/where that state is managed to allow for this. I'll make a separate PR for this once I figure things out.

[sc-11706]
[sc-11738]

### CODE REVIEW

File: client/src/components/PortfolioFilters.tsx [hunk 1]
```diff
@@ -101,92 +104,111 @@ export const PortfolioFilters = React.memo(
...
+              selectedValues={zipSelections}
+              displayValue="name"
+              placeholder={i18n._(t`Search`) + `... (${zipOptions.length})`}
+              onApply={onZipApply}
+              onFocusInput={() => helpers.scrollToBottom(".mobile-wrapper-dropdown")}
+              avoidHighlightFirstOption={true}
+            />
+          </FilterAccordion>
+        </FiltersWrapper>
+        {[rsunitslatestActive, ownernamesActive, unitsresActive, zipActive].includes(true) && (
```
Comment (shakao) [inline client/src/components/PortfolioFilters.tsx:196]: could this just be a big "or" statement? `(rsunitslatesActive || ownernamesActive || ...)` probably not a huge diff but might be marginally more performant than the includes

File: client/src/components/PortfolioFilters.tsx [hunk 2]
```diff
@@ -234,6 +256,60 @@ export const PortfolioFilters = React.memo(
...
+              <span className="active-filter-count">{activeFilters}</span>
+            )}
+            {isOpen ? <CloseIcon className="closeIcon" /> : <ChevronIcon className="chevronIcon" />}
+          </summary>
+          <div className="dropdown-container scroll-gradient mobile-wrapper-dropdown">
+            {children}
+            {!!activeFilters && (
+              <button onClick={() => setIsOpen(!isOpen)} className="button is-primary">
+                <Trans>View Results</Trans>
+                {resultsCount && <span className="view-results-count">{resultsCount}</span>}
```
Comment (shakao) [inline client/src/components/PortfolioFilters.tsx:303]: this will evalutate to false when resultsCount is zero, do we want to show the "0" ?

**Review by shakao — APPROVED** [review]

File: client/src/components/PortfolioFilters.tsx (hunk 2 above)
Comment (austensen) [reply client/src/components/PortfolioFilters.tsx]: oh right, thanks for catching!

File: client/src/components/PortfolioFilters.tsx (hunk 1 above)
Comment (austensen) [reply client/src/components/PortfolioFilters.tsx]: ah good to know, yeah that is just simplier all around not sure what I was doing here.

File: client/src/components/PortfolioFilters.tsx [hunk 3]
```diff
@@ -101,92 +104,111 @@ export const PortfolioFilters = React.memo(
...
-              onClick={updateRsunitslatest}
-              className="filter filter-toggle"
-            >
-              <div className="checkbox">{rsunitslatestActive && <CheckIcon />}</div>
+        <FiltersWrapper
+          isMobile={isMobile}
+          activeFilters={activeFilters}
+          resultsCount={filteredBuildings}
+        >
+          {" "}
```
[the remark this answers is not in the document]
Comment (austensen) [reply client/src/components/PortfolioFilters.tsx]: thanks yeah this shouldn't be there!

File: client/src/components/PortfolioFilters.tsx [hunk 4]
```diff
@@ -101,92 +104,111 @@ export const PortfolioFilters = React.memo(
...
     };
 
+    const activeFilters = [rsunitslatestActive, ownernamesActive, unitsresActive, zipActive].filter(
+      Boolean
+    ).length;
+
     return (
       <div className="PortfolioFilters" ref={ref}>
-        <div className="filter-for">
+        <div className="new-container">
```
[the remark this answers is not in the document]
Comment (austensen) [reply client/src/components/PortfolioFilters.tsx]: ah yeah good point, I changed this one for now, but will later do a pass over them all

File: client/src/components/Multiselect.tsx [hunk 5]
```diff
@@ -30,6 +30,7 @@ export interface IMultiselectProps {
   onSearch?: (value: string) => void;
   onKeyPressFn?: (event: any, value: string) => void;
   onApply: (selectedList: any) => void;
+  onFocusInput: () => void;
```
[the remark this answers is not in the document]
Comment (austensen) [reply client/src/components/Multiselect.tsx]: right yeah, should be optional

**Pull request merged**